src/models.py

```diff
@@ -981,7 +992,15 @@ def _tagger_forward(self, batch, task, predict):
             logits = logits.view(b_size * seq_len, -1)
             out['logits'] = logits
             targs = batch['targs']['words'][:, :seq_len].contiguous().view(-1)
-
+        if "mask" in batch:
+            # prevent backprop for tags generated for tokenization-introduced tokens
```

Comment:
Why not just mask out the loss for these positions, so the gradients are zero?

Reply:
Yep that's also definitely another way to do this!

Reply:
Ok. If going with a gather/select approach, can you add a comment to that extent, something like: "select only the targets that correspond to original tags, ignoring the introduced placeholder tags"